src/api/m64p_types.h

```diff
@@ -224,11 +224,11 @@ typedef enum
 
 typedef enum
 {
-    SAVETYPE_EEPROM_4KB = 0,
-    SAVETYPE_EEPROM_16KB,
+    SAVETYPE_EEPROM_4K = 0,
```

Comment:
KB is not accurate; the EEPROM sizes are in kilo*bits*: http://n64devkit.square7.ch/pro-man/pro26/26-06.htm

>There are two types of EEPROM according to their capacities. One is the 4k-EEPROM. The capacity of this type is 4 kilobits (64 words * 64 bits = 512 bytes). The other is the 16k-EEPROM. The capacity of this type is 16 kilobits (256 words * 64 bits = 2048 bytes).